src/app/app.html

```diff
@@ -1,14 +1,19 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title aria-label="Peeveetrack">
+          PVTrAck
+          <button ion-button menuClose style="float: right;" aria-label="Close the menu" tabindex=2>
+            <ion-icon name="close"></ion-icon>
+          </button>
+        </ion-title>
       </ion-toolbar>
     </ion-header>
 
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
-        <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <button role="link" ion-item *ngFor="let p of pages" (click)="openPage(p)">
```

Comment:
Does removing `menuClose` mean that the menu will stay open when I click on a menu item and am brought to the page I selected?